### DISCUSSION TITLE
What is undici

### DISCUSSION THREAD

**Discussion by ivanjeremic** [nodejs/undici#1302] [General]
I'm a bit of confused if undici is the right tool for me currently use axios is undici a similar tool that I can use to replace axios?

**Comment by mcollina** [comment]
If you are happy with Axios, there is no need to replace it. Undici is a replacement for `http.request`, which Axios uses underneath.

**Reply by kbariotis** [reply]
@mcollina isn't `fetch` that was recently added to Node.js the replacement for `http.request`? I'm a bit confused as well because this seems like an external library outside of Node.js core but being developed by Node.js developers? Apologies if I missed the memo :)

**Reply by kbariotis** [reply]
[Ok probably this should answer my question](https://nodejs.medium.com/introducing-undici-4-1e321243e007)

**Reply by kbariotis** [reply]
So why is it an external library? Is it to just avoid such big breaking changes?

**Reply by mcollina** [reply]
This library is being used by Node.js to implement fetch(). It's also developed by Node.js developers because we could not improve http.request without breaking everybody.
